examples/next/docs/vue3/demo/src/main.ts

```diff
@@ -0,0 +1,10 @@
+import Handsontable from 'handsontable';
+import { HotTable } from "@handsontable/vue3";
```

Comment:
In some places, there are single quites (`'`) being used. In others - the double quotes (`"`). 
Could you unify that? Most probably to the single quotes, as that's what we use in the Handsontable codebase. (with the exception of DOM-related stuff, like declaring props in the templates)